src/AppInstallerCLICore/pch.h

```diff
@@ -49,11 +49,9 @@
 
 #include <AppInstallerArchitecture.h>
 #include <AppInstallerDateTime.h>
-#include <AppInstallerDeployment.h>

```

Comment:
Is there a reason we specifically move these 2 out of pch to individual files?

Reply:
I've realized that even cross project it is wasteful to have our own files in the pch.  I didn't want to move everything and have the size of this change explode, so I moved the ones that I was touching to make my builds faster.